debugtools/DDR_VM/src/com/ibm/j9ddr/vm29/tools/ddrinteractive/commands/J9BCUtil.java

```diff
@@ -691,10 +692,12 @@ private static void dumpRomStaticField(PrintStream out, J9ROMFieldShapePointer r
 	}
 
 
+	private static final long J9AccClassIsUnmodifiableBit = J9ConstantHelper.getLong(J9JavaAccessFlags.class, "J9AccClassIsUnmodifiable", 0);
+
 	/*
-	 * Dump a printed representation of the specified @accessFlags to stdout.
-	 * Answer zero on success
+	 * Dump a printed representation of the specified @accessFlags to @out.
 	 */
+
```

Comment:
Since the comment applies to the method, it would be better if this blank line was removed.

Reply:
What is `@out`? What was wrong with the reference to 'stdout'?

Reply:
Based on the rest of the comment `@` refers to a parameter name, which is this case is called `out` - the code does not write to stdout explicitly.

Reply:
At this point, let's either leave this alone (the correct solution), or I'll just delete the comment - no other function has one, and the comment provides no real edification.

Reply:
Sorry, I should have looked at the method's parameters.